aten/src/ATen/native/BatchLinearAlgebra.cpp

```diff
@@ -3143,7 +3136,7 @@ Tensor linalg_svdvals(const Tensor& input) {
   // the singular vectors are not exposed to the user
   const bool input_requires_grad = (at::GradMode::is_enabled() && input.requires_grad());
   std::tie(std::ignore, singular_values, std::ignore) =
-      at::_svd_helper(input, /*some=*/input_requires_grad, /*compute_uv=*/input_requires_grad);
+      at::_svd_helper(input, /*some=*/true, /*compute_uv=*/input_requires_grad);
```

Comment:
Could you also check the `linalg_svdvals_out` function below? The second argument there says `/*full_matrices=*/false`, which seems different from `/*some=*/true` here.

Reply:
Yes, it should be `true` there. Thanks for spotting the problem.